src/language-css/loc.js

```diff
@@ -186,27 +184,21 @@ function replaceQuotesInInlineComments(text) {
 
       case "comment-inline":
         if (c === '"' || c === "'") {
-          inlineCommentContainsQuotes = true;
+          chars[i] = SPECIAL_CHAR_PLACEHOLDER;
+        }
+        if (c === "/" && text[i + 1] === "*") {
+          chars[i] = SPECIAL_CHAR_PLACEHOLDER;
+          chars[i + 1] = SPECIAL_CHAR_PLACEHOLDER;
+          i++;
         }
         if (c === "\n" || c === "\r") {
-          if (inlineCommentContainsQuotes) {
-            inlineCommentsToReplace.push([inlineCommentStartIndex, i]);
-          }
           state = "initial";
-          inlineCommentContainsQuotes = false;
         }
         continue;
     }
   }
 
-  for (const [start, end] of inlineCommentsToReplace) {
-    text =
-      text.slice(0, start) +
-      text.slice(start, end).replace(/["']/g, " ") +
-      text.slice(end);
-  }
-
-  return text;
+  return chars.join("");
```

Comment:
I'm worried about the efficiency of this approach. Previously, string manipulations happened only if bug-inducing comments were found. Now they will happen always, plus creating a potentially huge array with `text.split("")` looks memory-unfriendly. Would be good to explore the performance impact with `--debug-benchmark` and a big file.

Reply:
I think replace and concat text multiple times also slow.

How about a new empty string, concat the char when parsing? I'm not sure about performance.

Reply:
The thing is, often files don't contain such comments at all.

Reply:
```js
const string = '*'.repeat(1e7)

console.time('array');
const a = string.split('');
for (let i = 0; i< 1e4;i++) {
  a[i] = '_'
}
a.join('')
console.timeEnd('array');

console.time('replace');
for (let i = 0; i< 1e4;i++) {
  string.slice(0,i) + string.slice(i,1).replace(/\*/g,'_') + string.slice(i+1)
}
console.timeEnd('replace');
```

Yours faster...

Reply:
I'll restore yours

Reply:
You can't really get any meaningful results this way. Something like Benchmark.js needs to be used to run things in a loop. It's even built into Prettier (`--debug-benchmark`).

Anyway, my point is that for most files these string modifications aren't needed at all.